packages/web-extension/README.md

```diff
@@ -1,7 +1,287 @@
 # Cardano JS SDK | web-extension
 
-Web extension wallet utilities
+## About the Package Name
 
-e2e tests uses a pre-built [example dapp](https://github.com/input-output-hk/cardano-wallet-connector) and pretends to be "ccvault" wallet
+**Web-extension** can refer to a “browser extension” or the Web-Extensions browser API; it was chosen because the package is coupled to this API.
 
-![web-extension setup](./web-extension.jpg)
```

Comment:
Let's remove the file too if it's no longer relevant

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1052/commits/045b2e5c15f1897a7da5f0c03f440daa6e5a5aa3